WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginMagicLinkRequestFragment.java

```diff
@@ -94,9 +96,20 @@ public void onClick(View v) {
             }
         });
 
+        mAvatarProgressBar = view.findViewById(R.id.avatar_progress);
         WPNetworkImageView avatarView = (WPNetworkImageView) view.findViewById(R.id.gravatar);
-        avatarView.setImageUrl(GravatarUtils.gravatarFromEmail(mEmail, getContext().getResources()
-                .getDimensionPixelSize(R.dimen.avatar_sz_login)), WPNetworkImageView.ImageType.AVATAR);
+        avatarView.setImageUrl(GravatarUtils.gravatarFromEmail(mEmail, getContext().getResources().getDimensionPixelSize(R.dimen.avatar_sz_login)), WPNetworkImageView.ImageType.AVATAR,
+                new WPNetworkImageView.ImageLoadListener() {
+            @Override
+            public void onLoaded() {
+                mAvatarProgressBar.setVisibility(View.GONE);
+            }
+
+            @Override
+            public void onError() {
+                mAvatarProgressBar.setVisibility(View.GONE);
```

Comment:
I made the change with 9267290 and used a `grey_lighten_10` gridicon-user-circle as the default and error image for the gravatar.